api/v1/helpers/verbs/doDelete.js

```diff
@@ -37,8 +37,7 @@ function doDelete(req, res, next, props) {
   if (featureToggles.isFeatureEnabled(constants.featureName) &&
   props.modelName === 'Sample') {
     const sampleName = req.swagger.params.key.value.toLowerCase();
-    delPromise = u.getUserNameFromToken(req)
-      .then((user) => redisModelSample.deleteSample(sampleName, user));
+    delPromise = redisModelSample.deleteSample(sampleName, req.user);
```

Comment:
can req.user be undefined here? Samples controller goes directly to doDelete.

Reply:
The user object is always attached to the req by the verifyUserToken in jwtUtil. So, req.user will never be undefined.